panel/pane/image.py

```diff
@@ -440,13 +441,8 @@ def _transform_object(self, obj: Any) -> Dict[str, Any]:
         if self.encode:
             ws = f' width: {w};' if w else ''
             hs = f' height: {h};' if h else ''
-            data = f'<img src="{self._b64(data)}" style="max-width: 100%; max-height: 100%; object-fit: contain;{ws}{hs}"></img>'
-        elif self.width or self.height or self.sizing_mode not in (None, 'fixed'):
```

Comment:
I've read the warning below 10 times and still I do not understand it :-). I don't think what ever was ment with it applies after this fix.

Reply:
Yep indeed, your changes address the problem.